admin/class-monk-admin.php

```diff
@@ -1372,7 +1371,7 @@ public function monk_add_menu_translation_fields() {
 	 * @return void
 	 */
 	public function monk_change_nav_menu_fields() {
-		$screen   = get_current_screen()->base;
+		$screen = get_current_screen()->base;
```

Comment:
Create the method below and use it to validate every get_current_screen occurrence.
```
private function get_current_screen() {
    if ( function_exists( 'get_current_screen' ) ) {
        $screen = get_current_screen();

        if ( is_a( $screen, 'WP_Screen' ) && ! empty( $screen ) ) {
            return $screen;
        }
    }

    return false;
}
```

Use as:
```
$screen = $this->get_current_screen();

if ( $screen ) {
    // Your logic...
}
```